### ISSUE TITLE
Missing outils.js

### ISSUE THREAD

**Issue by bblumenfelder** [biblissima/collatinus-web-client#5]
The following resource is missing in the js-folder:
`<script type="text/javascript" src="/js/outils.js"></script>`

Is anything important inside that file?

**Comment by regisrob** [comment]
No, this js file is not important. Please note that this repo is not up-to-date compared to what is online at https://outils.biblissima.fr/fr/collatinus-web/. The collatinus-web html page is now included and managed in a larger Jekyll site that is not available on Github.
Could I ask what you would like to accomplish with this web page?

**Comment by bblumenfelder** [comment]
Thanks for your answer. 
I'm trying to build a local API on my dev machine that I can send requests to for morphological analysis. The daemon is running but I only get empty responses. I guess this is a configuration issue. I might open another issue for clarity reasons.
